src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -1063,30 +1065,24 @@ class RsInnerAttrStub(
 class RsMetaItemStub(
     parent: StubElement<*>?, elementType: IStubElementType<*, *>,
     val name: String?,
-    val hasEq: Boolean,
-    val value: String?
+    val hasEq: Boolean
 ) : StubBase<RsMetaItem>(parent, elementType) {
     object Type : RsStubElementType<RsMetaItemStub, RsMetaItem>("META_ITEM") {
         override fun createStub(psi: RsMetaItem, parentStub: StubElement<*>?): RsMetaItemStub =
-            RsMetaItemStub(parentStub, this, psi.name, psi.eq != null, psi.litExpr?.stringLiteralValue)
+            RsMetaItemStub(parentStub, this, psi.name, psi.eq != null)
```

Comment:
Did you intentionally remove `value` and not add `RsStubLiteralKind`?

Reply:
Yep
Now, literal expressions keep their values in stubs so no need to duplicate them - we can just get value from the corresponding literal expr